02/studenttests.ml

```diff
@@ -481,19 +518,64 @@ let instruction_tests = [
       && int64_of_sbytes (Gradedtests.sbyte_list m.mem (mem_size-8)) = Int64.sub Int64.max_int 254L
     )
   );
+  ("setb_cmp", Gradedtests.machine_test "1" 6 (set_cmp 10L 10L Eq)
```

Comment:
Can you use a description of the expected output as in the other test cases? Also I think some tests that follow contain typos in the description